packages/cli/generation/local-generation/local-workspace-runner/src/runLocalGenerationForWorkspace.ts

```diff
@@ -119,7 +124,25 @@ async function writeFilesToDiskAndRunGenerator({
     if (firstLocalOutputItem == null) {
         return;
     }
-    await rm(absolutePathToLocalOutput, { force: true, recursive: true });
+    // await rm(absolutePathToLocalOutput, { force: true, recursive: true });
+    const absolutePathToFernignore = path.join(absolutePathToLocalOutput, ".fernignore");
```

Comment:
@SlamChillz I could see us moving this into a separate function entirely called `copyGeneratedFiles()`

Reply:
So, I should move the entire logic from line 127 into function?